extras/ribbon.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * @license
+ * Copyright 2015 Petr Shevtsov (petr.shevtsov@gmail.com)
+ * MIT-licensed (http://opensource.org/licenses/MIT)
+ *
+ * Plugin options:
+ *
+ *  `data`: Array of numeric values (0-1) corresponding to the position
```

Comment:
Well, first of all, for example, data items could be somehow calculated (by some other different tools — e.g. R). So it would require some extra layer to process the appropriate mappings between calculations and actual colors. Also, data is data but color is representation.